### PR TITLE
QEA-1512: Add Element#jquery_click

### PR DESCRIPTION
**Pull request by rizzza**
React is proving to be a little bit troublesome with some element `click` interactions. Adding a `jquery_click` to get around those annoyances.

### CODE REVIEW

File: lib/page.rb [hunk 1]
```diff
@@ -95,6 +95,17 @@ def self.jquery_loaded?
...
     end
 
+    #
+    # JQuery click
+    # @param [String] CSS selector
+    # 
+    # Occasionaly selenium is unable to click on elements in the DOM which have some 
+    # interesting React goodies around the element. 
+    #
+    def self.jquery_click(selector)
```
Comment (yet-another-lucas) [inline lib/page.rb]: Why does the page have a click method?

Comment (rizzza) [inline lib/page.rb]: > Why does the page have a click method?

To be able to click without defining a element or page object.

Comment (yet-another-lucas) [inline lib/page.rb]: Ahh that makes sense, thanks.

**Comment by rizzza** [comment]
> Seeing failed tests on Jenkins on this branch.

Looks like an environment issue firing up everything. Passed on retry.
https://jenkinsmaster0001s1mdw1.sendgrid.net/job/QEA/job/gridium_test/4/console

**Pull request merged**